.gitignore

```diff
@@ -1,2 +1,3 @@
 /.history
-/target
\ No newline at end of file
+/target
+test_item
```

Comment:
Odd, later in this PR, you've fixed missing newline. Maybe you should drop an editorconfig file in this project to make sure editors (at least sensible ones) are consistent.

Maybe put something like the below in`/.editorconfig` (taken from rust project's editorconfig):
```
root = true

[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 4

[*.md]
# double whitespace at end of line
# denotes a line break in Markdown
trim_trailing_whitespace = false

[*.yml]
indent_size = 2
```

Reply:
That part related to to `*.md` files is very interesting one, I didn't know that.